src/Razor/test/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test.Common/Extensions/SourceTextExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test.Common.Extensions
+{
+    internal static class SourceTextExtensions
+    {
+        public static void GetLinesAndOffsets(
```

Comment:
We have a similar extension method today but it takes in a `TextSpan` instead of `Span` (which surprisingly don't inherit from one another 😕)